### ISSUE TITLE
Bad count item in the cart button

### ISSUE THREAD

**Issue by moafred** [Sylius/Sylius#12064]
**Sylius version affected**: 1.8.4

**Description**  
The number of items in the cart displayed in the header button is not well calculated.
It count the number of item without taking in account the quantity.

**Steps to reproduce**  
Add more than one item in the cart.

**Possible Solution**  
I was able to fix it in https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Widget/_button.html.twig#L7.
```diff
-{{- 'sylius.ui.item.choice'|trans({'%count%': cart.items|length}) }}
+{{- 'sylius.ui.item.choice'|trans({'%count%': cart.countItemsForCart}) }}
```
And in my Order Entity:
```php
public function countItemsForCart(): int
{
    $count = 0;
    foreach ($this->items as $item) {
        $count += $item->getQuantity();
    }

    return $count;
}
```
The `countItems` of `Sylius\Component\Core\Model\Order` also count without the quantity.

**Comment by coldic3** [comment]
Hey @moafred,
I see why it can be misleading but I'm not sure we should change it. @CoderMaggie can you add your opinion?

**Comment by GSadee** [comment]
Closing as it has been fixed in #14207
